seatunnel-connectors-v2/connector-file/connector-file-base/src/test/resources/excel/test_read_excel.conf

```diff
@@ -18,6 +18,7 @@
 {
   sheet_name = "Sheet1"
   skip_header_row_number = 1
+  date_format = "yyyy/M/d"
```

Comment:
Is that we must add `date_format` config to pass ci?

Reply:
> Is that we must add `date_format` config to pass ci?
It is not necessary to use date_format, datetime_format, time_format, see LocalFile
These three parameters exist in the parameters, allowing users to set them.

https://seatunnel.apache.org/docs/2.3.8/connector-v2/source/LocalFile

 In the previous version, I did not see the use of configuration. In this modification, the parameters are used. If date_format, datetime_format, time_format is configured, it will be based on the configured date_format. datetime_format,t ime_format performs time format parsing. If it is not configured, the cell format will be used to obtain the original value of the stored double type. If the cell format is not configured as a time type, DateTimeUtils.matchDateTimeFormatter / DateUtils.matchDateFormatter  will be used to format the string with time and date. Match is used to parse

Reply:
so revert this config can pass ci too?

Reply:
> so revert this config can pass ci too?

yes

Reply:
ok